packages/babel-plugin-polyfill-corejs3/test/fixtures/regression/preset-env-compatibility/output.mjs

```diff
@@ -0,0 +1,291 @@
+import "core-js/modules/es.symbol";
```

Comment:
These are without extension because they are injected by the old `preset-env` version, not by this plugin.